datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxUpgradeController.java

```diff
@@ -39,6 +40,9 @@
     @Inject
     private SdxUpgradeRecoveryService sdxUpgradeRecoveryService;
 
+    @Inject
+    private SdxService sdxService;
```

Comment:
this is not used

Reply:
Right, thanks. It might make sense to add a unit test checking that, the same we have for core.

Reply:
Asked it in the dev_internal.